src/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import App from './App';
+// eslint-disable-next-line
+import store from "./redux/configureStore";
```

Comment:
- [ ] As requested by @Gambit142 in the index.js file the correct path should be: "./redux/ConfigureStore". In your path for the import statement, you are using lowercase c instead of uppercase C for the configure store word. Please 🙏🏽 correct the path to avoid compilation errors.

- [ ] As requested by @Gambit142 kindly remove the Eslint-disable statement from index.js file. The Eslint helps us write clean, maintainable, and scalable codes. Using Eslint-disable statements defeats this purpose.